athenet/layers/inception.py

```diff
@@ -0,0 +1,82 @@
+"""Inception Layer."""
+
+import numpy as np
+
+import theano
+
+from athenet.layers import Layer, ConvolutionalLayer, ReLU, MaxPool, \
+    Concatenation
+
+
+class InceptionLayer(Layer):
+    def __init__(self, n_filters, input_layer_name=None, name='inception'):
+        """Create inception layer.
+
+        :n_filters: List of length 6: number of filters in convolutional
+                    layers.
+        """
```

Comment:
Insufficient comments. Consider @martacieslak comment style. It concerns all the comments.